website/source/docs/guides/deployment.html.md

```diff
@@ -26,11 +26,18 @@ The following instance configurations are recommended.
 |Large|4-8 core|32-64+ GB RAM|100 GB|**AWS**: m5.2xlarge, m5.4xlarge|
 |||||**Azure**: Standard\_D4\_v3, Standard\_D5\_v3|
 |||||**GCE**: n1-standard-32, n1-standard-64|
+|XL|12-24 core|64+ GB RAM|SSD|**AWS**: m5d.4xlarge|
+|||||**Azure**: Standard\_D16\_v3, Standard\_D32\_v3|
+|||||**GCE**: n1-standard-32, n1-standard-64|
 
 The **small size** instance configuration is appropriate for most initial production deployments, or for development/testing environments. The large size is for production environments where there is a consistently high workload. Suggested instance types are provided for common platforms, but do refer to platform documentation for up-to-date instance types that align with the recommended resources.
 
 ~> **NOTE** For large workloads, ensure that the disks support a high number of IOPS to keep up with the rapid Raft log update rate.
 
+The recommended size with the default configuration for a single datacenter is 5,000 nodes. Consul can support larger single datacenter cluster sizes by tuning the [gossip parameters](/docs/agent/options.html#gossip_lan) and ensuring Consul agents -- particularly servers -- are running on sufficient hardware. There are real production users of Consul running with greater than 25,000 nodes in a single datacenter today by tuning these parameters.
+
```

Comment:
Looks great. I think we could add to this " There are real production users of Consul running with greater than 25,000 nodes in a single datacenter today by tuning these parameters. The minimum hardware requirement must be XL (Link to above)  or better to achieve such scale.